examples/ts-environments/index.ts

```diff
@@ -12,3 +12,21 @@ var environment = new service.Environment("testing-environment", {
     myNumber: 1`
   )
 })
+
+// A tag that will always be placed on the latest revision of the environment
+var stableTag = new service.EnvironmentVersionTag("StableTag", {
+  organization: environment.organization,
+  environment: environment.name,
+  tagName: "stable",
+  revision: environment.revision
+})
+
+// A tag that will be placed on each new version, and remain on old revisions
+var versionTag = new service.EnvironmentVersionTag("VersionTag", {
+  organization: environment.organization,
+  environment: environment.name,
+  tagName: environment.revision.apply((rev: number) => "v"+rev),
+  revision: environment.revision
+}, {
+  retainOnDelete: true
+})
```

Comment:
You had mentioned yesterday that this resource worked without the outputs... but we're not really using outputs in any of these examples so they're not really tested. Something like `export const tag = versionTag.tagName;` would test it.

Reply:
Yeah, I realized that I never actually used any properties. I tested and checked that after adding to the schema the properties have appeared. (It's easy to check in Dotnet)